### PR TITLE
fix: replace finishedAt with startedAt in Argo

### PR DESCRIPTION
**Pull request by sam-robson**
Operations have always started, but not always finished.
https://github.com/RoadieHQ/roadie-backstage-plugins/issues/199

### CODE REVIEW

**Comment by Xantier** [comment]
Thanks for the contribution @sam-robson! I'll spin this up on our ArgoCD instances and take a look && merge eventually

File: plugins/backstage-plugin-argo-cd/src/types.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const argoCDAppDetails = t.type({
       status: t.string,
     }),
     operationState: t.type({
-      finishedAt: t.string,
+      startedAt: t.string,
```
Comment (Irma12) [inline plugins/backstage-plugin-argo-cd/src/types.ts:13]: Even though I have nothing against adding startedAt property, I would keep finishedAt in a form of:
finishedAt: t.union([t.string, t.undefined])
(The reason behind this is we use finishedAt in order to display 'Last synced' data) https://github.com/RoadieHQ/roadie-backstage-plugins/blob/main/plugins/backstage-plugin-argo-cd/src/components/ArgoCDDetailsCard.tsx

**Review by iain-b — APPROVED** [review]
@Irma12 Your comment was addressed so this seems good to go ?

**Comment by Irma12** [comment]
@sam-robson Thank you for the contribution once again 🎉

**Review by Irma12 — APPROVED** [review]

**Pull request merged**